cms/djangoapps/contentstore/views/item.py

```diff
@@ -1256,6 +1274,7 @@ def create_xblock_info(xblock, data=None, metadata=None, include_ancestor_info=F
             elif xblock.category == 'course':
                 xblock_info.update({
                     'highlights_enabled_for_messaging': course.highlights_enabled_for_messaging,
+                    'discussion_enabled': xblock.discussion_enabled
```

Comment:
We don't seem to be returning the is_discussable flag for any XBlock. Please use the approach from https://github.com/edx/edx-platform/pull/24601/ to add this for all XBlocks that have the field.